tests/Operation/AggregateFunctionalTest.php

```diff
@@ -103,12 +103,29 @@ public function testTypeMapOption(array $typeMap = null, array $expectedDocument
         $this->createFixtures(3);
 
         $pipeline = [['$match' => ['_id' => ['$ne' => 2]]]];
+
         $operation = new Aggregate($this->getDatabaseName(), $this->getCollectionName(), $pipeline, ['typeMap' => $typeMap]);
         $results = iterator_to_array($operation->execute($this->getPrimaryServer()));
 
         $this->assertEquals($expectedDocuments, $results);
     }
 
+    public function testTypeMapInline()
+    {
+        if (version_compare($this->getServerVersion(), '3.6.0', '>=')) {
+            $this->markTestSkipped('Inline aggregations are not supported');
+        }
+
+        $this->createFixtures(3);
+
+        $pipeline = [['$match' => ['x' => 1]], ['$out' => $this->getCollectionName() . '.output']];
```

Comment:
The `$out` operator implies that no results will be returned. I think it'd make more sense to rewrite this test case to copy the logic in `testTypeMapOption()` and add the `instanceof` check. Its data provider can also be re-used as-is.

This test can then be renamed `testTypeMapOptionWithoutCursor()` or something to that effect.